src/App.css

```diff
@@ -1,38 +1,63 @@
-.App {
-  text-align: center;
+.container {
+  align-items: center;
+  padding: 0px;
+  background: #181818;
+}
+.topbar {
+  width: 1097px;
+  height: 119px;
```

Comment:
You don't have to give the width here. Instead, you can apply a max-width to the nav. And use a round figure whenever applying CSS.